dockstore-common/src/test/resources/fixtures/dockstore12.yml

```diff
@@ -10,6 +10,11 @@ workflows:
      primaryDescriptorPath: /Dockstore.cwl
      testParameterFiles:
        - /dockstore.cwl.json
+  # A workflow with no name:
+  -  subclass: cwl
```

Comment:
If "CWL" works, use "CWL"

Reply:
Maybe it should just be case in-sensitive. 
I don't think we're ever going to have workflow languages that are distinguishable only by taking into account case.

Reply:
It is case-insensitive -- the value ultimately gets fed to [this line](https://github.com/dockstore/dockstore/blob/637669474d6f7d2799e796ac0bb1e0337e8b575e/dockstore-common/src/main/java/io/dockstore/common/DescriptorLanguage.java#L156).

I'm inclined to leave it as is for now, at least for this PR (and there are already existing tests with lowercase "cwl").

Right now the subclass is a string, maybe we should make it an enum like we did for the subclass in services, but I think it gives us more flexibility to leave it as a string.

Reply:
It may be case insensitive here, but there are far too many places on the backend that are case sensitive.  See (Workflow descriptorType, ToolDescriptor type, etc ).  It's already a real pain dealing with close to 5 sets of descriptor type enums, don't want to potentially deal with another one.

It is case-insensitive, great! But like i said:

If "CWL" works, use "CWL".

Reply:
I think there is a distinction to be made here. 
I agree that as developers, we probably want to pick one and in parts of the code only we really touch, sure.
However, the `.dockstore.yml` is something users will need to create and I really don't want to tell users to switch their case. If anything, I think there should be two copies of this file to nail down that this really is intended to be case insensitive.

Reply:
This file is a fixture. It already has two other workflows in it, with subclasses 'wdl' and 'cwl'. I left those I changed this one to be 'CWL' and added a test that ensures that both work.